frontend/views/components/graphs/Overview.vue

```diff
@@ -7,7 +7,7 @@ div
         :amount='withCurrency(totalCovered)'
         color='warning-solid'
         variant='inline'
-      ) {{ L('Total covered') }}
+      ) {{ L('Total needed') }}
```

Comment:
Why was this change made? It seems wrong, resulting in two "Total needed":

<img width="598" alt="Screen Shot 2022-03-15 at 5 54 27 PM" src="https://user-images.githubusercontent.com/138706/158495805-b81db83a-1718-4535-9b2b-6c5386309941.png">

Reply:
Request from @leihla ?

Reply:
@pieer I asked @leihla about this just now and she said that the conclusion on the Monday call was to revert it back to what it was, i.e. leave it as `Total covered`.